resources/app/src/components/people/DemographicsForm.vue

```diff
@@ -933,7 +945,44 @@ export default {
             if (this.isFormValid) {
                 alert('Form is valid and ready to be submitted!');
                 items = {
-                    ...this.formdata,
+                  //  ...this.formdata,
+                  birth_country: this.formdata.birth_country,
+                    birth_country_other: this.formdata.birth_country_other,
+                    reside_country: this.formdata.reside_country,
+                 reside_country_other: this.formdata.reside_country_other,
+        state: this.formdata.state,
```

Comment:
speaking of leaving things out, the variable above is `reside_state`, not `state`, so the object won't end up getting the field value...

Reply:
I resolved this issue before seeing this comment.